### PR TITLE
DimSI-1.0-goolf-1.4.10.eb

### PR DESCRIPTION
**Pull request by pescobar**

### CODE REVIEW

**Comment by fgeorgatos** [comment]
Test report by @fgeorgatos
**SUCCESS**
Build succeeded for 1 out of 1
Linux debian 6.0.10, Intel(R) Xeon(R) CPU           L5640  @ 2.27GHz, Python 2.6.6
See https://gist.github.com/5853015fc451dd692aaa for a full test report.

File: easybuild/easyconfigs/d/DimSI/DimSI-1.0-goolf-1.4.10.eb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+premakeopts += '$CC $OPTFLAGS DimSI_binomial.c -lm -o binomial && '
+
+# with this hack we avoid the "make" command from MakeCp block to be executed
+premakeopts += 'echo compilation done. Avoiding to run make '
+
+files_to_copy = [(["Poisson", "binomial"], "bin")]
+        
+sanity_check_paths = {
+    'files': ["bin/Poisson", "bin/binomial"],
+    'dirs': [""],
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/d/DimSI/DimSI-1.0-goolf-1.4.10.eb]: drop "", remove last empty line.

@boegel: only style remarks, test/merge?

File: easybuild/easyconfigs/d/DimSI/DimSI-1.0-goolf-1.4.10.eb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+source_urls = ['http://webpages.uidaho.edu/hohenlohe/']
+sources = ['%(name)s_source_code.zip']
+
+start_dir = 'DimSI_source_code'
+
+# we run the compilation here. Lines like this:
+# gcc -O2 -march=native DimSI_Poisson.c -lm -o Poisson
+premakeopts = '$CC $OPTFLAGS DimSI_Poisson.c -lm -o Poisson && '
+premakeopts += '$CC $OPTFLAGS DimSI_binomial.c -lm -o binomial && '
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/d/DimSI/DimSI-1.0-goolf-1.4.10.eb]: prebuildopts?

File: easybuild/easyconfigs/d/DimSI/DimSI-1.0-goolf-1.4.10.eb (hunk 1 above)
Comment (fgeorgatos) [inline easybuild/easyconfigs/d/DimSI/DimSI-1.0-goolf-1.4.10.eb]: lgtm , minor changes needed here, put this in REVIEW plz!

**Comment by migueldiascosta** [comment]
@pescobar, this PR is being closed for the following reason(s): no activity for > 6 months.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**